pipeline_dp/pipeline_operations.py

```diff
@@ -74,3 +86,48 @@ def sample_fixed_per_key(self, col, n: int, stage_name: str):
 
   def count_per_element(self, col, stage_name: str):
     return col | stage_name >> combiners.Count.PerElement()
+
+
+class SparkOperations(PipelineOperations):
+  """Apache Spark Core adapter."""
+
+  def map(self, rdd, fn, stage_name: str = None):
+    return rdd.map(fn)
+
+  def map_tuple(self, rdd, fn, stage_name: str = None):
+    return rdd.map(fn)
+
+  def map_values(self, rdd, fn, stage_name: str = None):
+    return rdd.mapValues(fn)
+
+  def group_by_key(self, rdd, stage_name: str = None):
+    """Group the values for each key in the RDD into a single sequence.
+
+    Args:
+      rdd: input RDD
+      stage_name: not used
+
+    Returns:
+      An RDD of tuples in which the type of the second item
+      is the pyspark.resultiterable.ResultIterable.
+
+    """
+    return rdd.groupByKey()
+
+  def filter(self, rdd, fn, stage_name: str = None):
+    return rdd.filter(fn)
+
+  def keys(self, rdd, stage_name: str = None):
+    return rdd.keys()
+
+  def values(self, rdd, stage_name: str = None):
+    return rdd.values()
+
+  def sample_fixed_per_key(self, rdd, n: int, stage_name: str = None):
```

Comment:
This function is not just a wrapper of PySpark function, could you please write some simple test to it?

Reply:
Done.